example_cluster/setup_fake_config.py

```diff
@@ -0,0 +1,57 @@
+import json
+import os
+import shutil
+import socket
+import subprocess
+import sys
+from distutils.dir_util import copy_tree
+
+
+def find_open_ports():
+    for port in range(10000, 1, -1):
+        with socket.socket(socket.AF_INET, socket.SOCK_STREAM) as sock:
+            if sock.connect_ex(('localhost', port)) != 0:
+                return port
+
+
+def main():
+
+    print('-------------------------------------------------------')
+    print('Please run echo $$PAASTA_SYSTEM_CONFIG_DIR to continue')
+    print("Please set environment variable PAASTA_TEST_CLUSTER to the cluster you want to use.")
+    print("This is necessary for tron jobs")
+    print('-------------------------------------------------------')
+    cluster = os.environ.get('PAASTA_TEST_CLUSTER', 'norcal-devc')
+    config_path = 'fake_config'
+
+    copy_tree('/etc/paasta', config_path)
+    # Generate tron.json
+    tron_config = {'tron': {'url': f'http://tron-{cluster}:8089'}}
+    with open(config_path + '/tron.json', 'w') as f:
+        json.dump(tron_config, f)
+    # find unused port
+    port = find_open_ports()
+    # Generate api endpoints
+    api_endpoints = {'api_endpoints': {cluster: f'http://localhost:{port}'}}
+    os.remove(config_path + '/api_endpoints.json')
```

Comment:
You can just overwrite the file, do you have to bother removing it first?

Reply:
Since I copied the file from etc/paasta, I'll need to chmod if I don't remove.